.github/actions/markPullRequestsAsDeployed/markPullRequestsAsDeployed.js

```diff
@@ -50,17 +127,30 @@ message += `\n🍎 iOS 🍎|${iOSResult} \n🕸 web 🕸|${webResult}`;
  * @returns {Promise<void>}
  */
 function commentPR(pr) {
-    return GithubUtils.createComment(context.repo.repo, pr, message)
+    return getPRMessage(pr).then(message => GithubUtils.createComment(context.repo.repo, pr, message)
         .then(() => {
             console.log(`Comment created on #${pr} successfully 🎉`);
         })
         .catch((err) => {
             console.log(`Unable to write comment on #${pr} 😞`);
             core.setFailed(err.message);
+        }));
+}
+
+const run = function () {
```

Comment:
Not too sure about run's usage here. Just copied from another action. Please let me know if we don't need it.

Reply:
The point of creating a `run` function is so that you can import it into a unit test.

Reply:
But you can leave it like this. It's consistent with our other actions and the extra function isn't doing any harm